### DISCUSSION TITLE
3.5.0 版本发布 - 支持 ChatGPT 流式对话

### DISCUSSION THREAD

**Discussion by wzpan** [wzpan/wukong-robot#230] [官方通知]
本版本重点支持了 ChatGPT 流式对话和代码高亮，demo 详见这个视频：

【wukong-robot + ChatGPT 实现支持流式对话的智能音箱】 https://www.bilibili.com/video/BV1Bh411g7t2/?share_source=copy_web&vd_source=84877ef432f0e2745d4600027059e155

## features

* 极客模式的默认结束时间从5回合拉长到20回合 ([4a5524d](https://github.com/wzpan/wukong-robot/commit/4a5524d24dabcf52738a34314373c141e0be3ffa))
* 美化气泡排版，实现代码高亮 ([63d0e78](https://github.com/wzpan/wukong-robot/commit/63d0e7852908e56f3e80f14beb2726193a9027b1))
* 增加闲聊插件 ([c071590](https://github.com/wzpan/wukong-robot/commit/c071590827833e9e1c967e07100d82d6e9370abb)) 使用说明：https://wukong.hahack.com/#/official?id=gossip
* ChatGPT 支持流式输出，优化响应速度 ([53e969e](https://github.com/wzpan/wukong-robot/commit/53e969e690caa464b00d42cd1e84198f7ca8fd74))

## bug fixes

* 废弃activeMode([#224](https://github.com/wzpan/wukong-robot/issues/224));
* 修复极客模式下的若干bug ([5d79aca](https://github.com/wzpan/wukong-robot/commit/5d79aca50b9b2e68277176c96b758374c2ede46b))
* 移除多余的配置信息 ([349c97f](https://github.com/wzpan/wukong-robot/commit/349c97fabdb9cbd2f963b67cd502edaa4f7bb952))
* 支持长内容自动滚动，修复LocalPlayer播放问题 ([3121fb3](https://github.com/wzpan/wukong-robot/commit/3121fb3055f5f6554021773270500c6861cb1911))

**Comment by wangpingfan66** [comment]
请问一下可以使用中转API接入ChatGPT吗？比如NextChat
